Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG PARENT_VERSION=2.3.0-node20.15.0
+ARG PARENT_VERSION=2.5.0-node22.11.0	
 ARG PORT=3000
 ARG PORT_DEBUG=9229
 
```

Comment:
I know it was already there, but look like Matt added omit dev wrongly, so dev packages still included. On line 32 can you update to --omit=dev please (currently says --omit-dev)